app/src/main/java/de/westnordost/streetcomplete/quests/hairdresser/AddHairdresser.kt

```diff
@@ -0,0 +1,46 @@
+package de.westnordost.streetcomplete.quests.hairdresser
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddHairdresser : OsmFilterQuestType<Hairdresser>() {
+
+    override val elementFilter = """
+        nodes, ways with
+          (
+              shop = hairdresser
+              and !female and !male and !unisex
```

Comment:
I though the outcome of the discussion was that when unisex is specfied, it is ignored due to it being ambiguous and always re-asked?

Reply:
Hm, I asked which of the two choices you preferred in https://github.com/streetcomplete/StreetComplete/issues/4833#issuecomment-1464557177 and from your answer https://github.com/streetcomplete/StreetComplete/issues/4833#issuecomment-1483809818 I concluded that you prefer option (1) _"permanently ignore any hairdressers which have unisex=* tag set (thus preserving any ambiguous meaning that original mapper may have intended)"_ because _"otherwise we'd be fighting against iD windmills"_ (i.e. we'd be removing `unisex=yes` and adding `male=yes`+`female=yes`; whilst iD would be adding `unisex=yes` and removing `male=yes`+`female=yes` ? 

Or did I misunderstood and you prefer option (2) _"quest should be asked regardless of whether unisex=* is set, and when user has chosen current state of the hairdresser set male=* / female=* tags and remove unisex=* (thus fixing ambiguity as confirmed by new mapper)"_? 

Or did you want some third (unmentioned) option?

Reply:
The "agree to your statement" referred to 

> preferable to make situation unambiguous when we have the chance with mapper on the ground, rather then to simply "sweep the problem under the rug"

Regarding the remark about fighting against windmills, my hope would be that how iD currently handles these things would be checked and a PR/ticket be opened at iD to straighten this out (if necessary) in order that iD does not overwrite/delete unambiguous data with ambiguous data.

Reply:
Ok! I'll then implement `(2)`  tonight (and try to look into iD later)

Reply:
implemented replacing `unisex` tag. I'll lookup iD later (but it is not holding up this PR, which should be ready now)